src/components/pageLayout/pageHeader.js

```diff
@@ -10,26 +10,32 @@ import {
  *
  * @param {object} props
  * @param {Node} props.children
+ * @param {string} props.title
  * @returns {Node}
  */
-const PageHeader = ({ children }) => (
+const PageHeader = ({ children, title }) => (
   <RcsPageHeader>
-    <PageHeaderTitle title={children} />
+    {title && <PageHeaderTitle title={title} />}
```

Comment:
I made the title not required because we do have a situation in which we don't need the title, specifically, the Loader. The loader adds a graphical element into a title component when it could just be loading it directly into the PageHeader.

Reply:
k, well, looks like you'll need to update the loader too with the `title={<Skeleton...`